packages/slate-react/src/utils/slate-snapshot.js

```diff
@@ -0,0 +1,52 @@
+import closest from './closest'
```

Comment:
Could you rename this to `DomSnapshot` instead? as that seems more clear.

Reply:
Sure.

Reply:
Done